src/components/setup-sharehouse/DefaultCategory.ts

```diff
@@ -0,0 +1,76 @@
+import { ICategory } from '@/types/commons';
+import { randomUUID } from 'crypto';
+
+export const DefaultCategory: ICategory[] = [
+  {
+    id: randomUUID(),
+    name: 'Kitchen',
+    tasks: [
+      {
+        id: randomUUID(),
+        title: 'Clean the sink',
+        description:
+          '<li><ul>Scrub the sink with a sponge and bathroom cleaner. <u>Rinse thoroughly.</u></ul></li>',
```

Comment:
- The <li> element should be inside a <ul> or <ol>.
- modify it is always wrapped in a <p> tag.
* TipTap editor add `<p></p>` tag to every element.

`<p><ul><li>Scrub the sink with a sponge and bathroom cleaner. <u>Rinse thoroughly.</u></li></ul></p>`

Reply:
It cause a blank line. I think we don't need <p> before <li>

Reply:
Okay thanks!